src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -23,6 +23,8 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == '$(AndroidTargetFramework)'">
+    <TargetFrameworkIdentifier>MonoAndroid</TargetFrameworkIdentifier>
```

Comment:
I see build errors without these explicitly specified.  I get this sometimes on UWP and occasionally on iOS, too.